draft-ietf-rats-corim.md

```diff
@@ -100,9 +100,11 @@ normative:
 
 informative:
   RFC7942:
+  RFC9277:
```

Comment:
is this ref used?

Reply:
Not since I changed to cbor-record. Will remove.